src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
```

Comment:
Ensure that the element with the class 'tree' exists in the DOM before attaching event listeners or manipulating it. This will prevent potential runtime errors if the element is not found.